src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,32 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.health = health;
+    this.name = name;
+    this.hidden = false;
+    Animal.alive.push(this);
```

Comment:
Only Herbivore has a method of hide)
```suggestion
    this.name = name;
    Animal.alive.push(this);
```

Reply:
Fixed!